docs/se/guides/07_db_client.adoc

```diff
@@ -0,0 +1,604 @@
+TCP server running at tcp://127.0.1.1:9092 (others can connect)
+PG server running at pg://127.0.1.1:5435 (only local connections)
+----
+
+Open the console at http://127.0.1.1:8082 in your favorite browser. It displays a login window.
+Select `Generic H2 (Server)` from `Saved Settings`. Here is the list of setting, all of them should be set by default.
+
+* Driver Class: org.h2.Driver
+* JDBC URL: jdbc:h2:tcp://localhost/~/test
+* User Name: sa
+* Password:
+
+Password must stay empty. Press connect, the browser displays a web page. The database is correctly set and running.
+
+=== Create a sample SE project
+
+Generate the project sources using the Helidon SE Maven archetype.
+The result is a simple project that can be used for the examples in this guide.
+
+[source,bash,subs="attributes+"]
+.Run the Maven archetype:
+----
+mvn -U archetype:generate -DinteractiveMode=false \
+    -DarchetypeGroupId=io.helidon.archetypes \
+    -DarchetypeArtifactId=helidon-quickstart-se \
+    -DarchetypeVersion={helidon-version} \
+    -DgroupId=io.helidon.examples \
+    -DartifactId=helidon-quickstart-se \
+    -Dpackage=io.helidon.examples.quickstart.se
+----
+
+A new directory named `helidon_quickstart-se` is created.
+
+[source,bash]
+.Enter into this directory:
+----
+cd helidon_quickstart-se
+----
+
+=== Add new dependency
+
+First thing to do is to open the pom.xml file and add helidon dependencies to use the DB Client.
+
+[source,xml]
+.Copy those dependencies:
+----
+<dependency>
+    <groupId>io.helidon.dbclient</groupId>       <1>
+    <artifactId>helidon-dbclient</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.dbclient</groupId>      <2>
+    <artifactId>helidon-dbclient-jdbc</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.integrations.db</groupId>   <3>
+    <artifactId>h2</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.dbclient</groupId>      <4>
+    <artifactId>helidon-dbclient-health</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.dbclient</groupId>      <5>
+    <artifactId>helidon-dbclient-metrics</artifactId>
+</dependency>
+<dependency>
+    <groupId>io.helidon.dbclient</groupId>      <6>
+    <artifactId>helidon-dbclient-jsonp</artifactId>
+</dependency>
+----
+<1> DB Client API dependency.
+<2> Using JDBC driver for this example.
+<3> H2 driver dependency.
+<4> Support for health check.
+<5> Support for metrics.
+<6> Support for Jsonp.
+
+=== Configure the DB Client
+
+To configure the application, helidon uses by default the application.yaml. The DB client configuration can be joined
+in the same file. It is located here: `src/main/resources`.
+
+[source,yaml]
+.Copy those properties into application.yaml
+----
+db:
+  source: jdbc  # <1>
+  connection:   # <2>
+    url: "jdbc:h2:tcp://localhost:9092/~/test"
+    username: "sa"
+    password:
+  statements:   # <3>
+    #Ping statement required for health check
+    ping: "SET @HELIDON_PING=0"   # <4>
```

Comment:
Ping is already removed from DB Client. We now have a separate configuration for Health Checks.